appcelerator/titanium_mobile_windows
# [TIMOB-19865] Revisit unit tests

Fix for [TIMOB-19886](https://jira.appcelerator.org/browse/TIMOB-19886) and [TIMOB-19865](https://jira.appcelerator.org/browse/TIMOB-19865).

- Some unit tests in NMocha failed on Windows Store app. Make sure all tests pass for both Windows Phone and Store.
- Remove unused capability
- Add test for Picker and 2DMatrix
- Re-enable some tests

How to reproduce:

- Create NMocha project for Windows Store app
```
$ node YOUR_titanium_mobile_windows/Tools/Scripts/generate_project.js -p WindowsStore new NMocha
```
- Open NMocha.WindowsStore.Win32/NMocha.sln and run it in Local Machine
- Results are printed out to the VS console, make sure all unit tests pass
